### DISCUSSION TITLE
[ADD]: Addition of DCO

### DISCUSSION THREAD

**Discussion by AbhijithGanesh** [SeaQL/sea-orm#643] [Ideas]
## Summary

It'd be good if we implement DCO (Developer certificate of Origin) for more accountability.

## Motivation

https://github.com/apps/dco

## Additional Information
N/A

**Comment by tyt2y3** [comment]
Thank you for the suggestion. Can you also perhaps name a few open source projects currently adopting this?

**Comment by AbhijithGanesh** [comment]
https://github.com/accordproject/web-components
